### PR TITLE
Store statement's property created & stored as DateTime instead of timestamp

### PR DESCRIPTION
**Pull request by Lctrs**
Since Doctrine supports handle of `\DateTime` instance, let's just use that.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+master
+------
```
Comment (xabbuh) [inline CHANGELOG.md]: 0.4.0 isn't released yet. So this change should be part of it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+
+* store statement's created and stored property as `\DateTime` instead of
+  `int` timestamp.
```
Comment (xabbuh) [inline CHANGELOG.md]: I would reword this to something like "the created and stored properties of the internal mapping `Statement` class are now instances of PHP's `\DateTime` class instead of integers representing UNIX timestamps".

**Comment by Lctrs** [comment]
@xabbuh Done.
